### DISCUSSION TITLE
[Feature request] Time-Series Bar Chart Custom Ordering

### DISCUSSION THREAD

**Discussion by davidjez** [apache/superset#18673] [Ideas]
**Is your feature request related to a problem? Please describe.**
Would it be possible to add sort_by in Time-Series Bar Chart, too?

Example: https://stackoverflow.com/questions/65588033/superset-time-series-bar-chart-how-to-order-metrics-as-they-appear-in-the-metri

Similar issues has been proposed here, but it is applicable to Bart Charts, only: https://github.com/apache/superset/issues/9998

**Comment by geido** [comment]
Hello @davidjez thanks so much for this idea. Just want to inform you that from now on we would like to use the issues only for true bugs. For that reason, I am moving this one to Discussions. Thanks!
